README.md

```
@@ -1,21 +1,34 @@
-# cmux-companion
+# merry
 
-iPhone companion app for [cmux](https://github.com/manaflow-ai/cmux) — mirror notifications, control workspaces, navigate surfaces, and send voice commands from your phone.
+iPhone companion for AI coding agents — mirror notifications, control
+workspaces, navigate surfaces, paste photos and files, and send voice commands
+from your phone. Fronts [cmux](https://github.com/manaflow-ai/cmux) and
+[herdr](https://herdr.dev), together or on their own.
 
 ## Architecture
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)